pkg/apb/types.go

```diff
@@ -200,3 +201,10 @@ type RecoverStatus struct {
 	InstanceID uuid.UUID `json:"id"`
 	State      JobState  `json:"state"`
 }
+
+// ExecutionContext - Contains the information necessary to track and clean up an APB run
+type ExecutionContext struct {
```

Comment:
I could do that (add PodName adn ServiceAccount to the context struct), though we would have 2 unrelated Context in the same scope, which may be confusing. Nothing against it really though, just was trying not to change too many existing structures.